playbooks/roles/capsule/tasks/install.yml

```diff
@@ -44,6 +45,11 @@
     src: "/tmp/{{ inventory_hostname }}{{ capsule_certs_tar }}"
     dest: ~/
 
+- name: 'Change cert permissions'
+  file: path='/etc/pki/katello/private' mode=0775
+  delegate_to: "{{ capsule_server }}"
+  when: devel == true
```

Comment:
@ehelms you already asked :smile: [here](https://git.io/vitJx) Its just something that we have to do in dev currently